htdocs/modules/system/class/thumbs/phpthumb.functions.php

```diff
@@ -784,8 +784,8 @@ static function SafeURLread($url, &$error, $timeout=10, $followredirects=true) {
 			curl_setopt($ch, CURLOPT_HEADER, false);
 			curl_setopt($ch, CURLOPT_RETURNTRANSFER, true);
 			curl_setopt($ch, CURLOPT_BINARYTRANSFER, true);
-			curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, false);
-			curl_setopt($ch, CURLOPT_SSL_VERIFYPEER, false);
+			curl_setopt($ch, CURLOPT_SSL_VERIFYHOST, 2);    // changed for XOOPS
```

Comment:
For more info:
[The Most Dangerous Code in the World: Validating SSL Certificates in Non-Browser Software by Martin Georgiev](https://www.cs.utexas.edu/~shmat/shmat_ccs12.pdf)

[Beware of Unverified TLS Certificates in PHP](https://blog.sucuri.net/2016/03/beware-unverified-tls-certificates-php-python.html)